helper/resource/testing.go

```diff
@@ -537,7 +538,7 @@ func Test(t TestT, c TestCase) {
 	log.SetOutput(logWriter)
 
 	// We require verbose mode so that the user knows what is going on.
-	if !testing.Verbose() && !c.IsUnitTest {
+	if !gotesting.Verbose() && !c.IsUnitTest {
```

Comment:
Could we wrap this in the `testing` package we created? So we don't need to import both of them here, at least? Which avoids the confusing import name clash?

Reply:
yes no problem